### DISCUSSION TITLE
[QUESTION]

### DISCUSSION THREAD

**Discussion by AmmarBagharib** [python-telegram-bot/python-telegram-bot#3629] [Q&A]
### Issue I am facing

I'm new here, and I'm trying to create a Q&A Telegram group which consists of the following:

Users comes from the public, admins will be people with advanced knowledge on subject matter. Therefore:

- users can only send their questions, they can't send any other messages
- Strictly, only the group admins are allowed to send answers to the questions posted by users.

I was thinking the users could send their questions through a bot which then posts it on the main channel to minimise the chance of them answering questions of other users. Currently I have a similar group but instead of a conducive Q&A, users are answering one another's questions instead of the group admins, and answering questions becomes counter-productive.

Does anyone have any advice on how I should go about doing this?

### Operating System

iOS

### Version of Python, python-telegram-bot & dependencies

```shell
-
```

**Comment by Poolitzer** [comment]
I am sorry, this GitHub issue tracker is for issues or technical questions arising while programming with our library, not asking about generic advice.

Yes you can certainly realize that, either how you envision it or you could let the bot delete all messages from users which arent in their question thread, that way they can search the group if desired.

But we won't program the bot for you, sorry.
